src/Migrations/20230908151840_APITokenRemoveSaltyColumn.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace NodeGuard.Migrations
+{
+    /// <inheritdoc />
+    public partial class APITokenRemoveSaltyColumn : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "IsSalty",
+                table: "ApiTokens");
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<bool>(
+                name: "IsSalty",
+                table: "ApiTokens",
+                type: "boolean",
+                nullable: false,
+                defaultValue: false);
+        }
+    }
```

Comment:
I would say, lets merge #303, remove migrations of this PR and create a new "clean" one with all the new changes

Reply:
Perfect